### PR TITLE
GH-10792: Create empty checkout Lib

### PR DESCRIPTION
**Pull request by plabadie**
closes GH-10792

### CODE REVIEW

File: docs/libs/creating-lib.md [hunk 1]
```diff
@@ -207,7 +230,7 @@ Add the following scripts:
 "release:myaccount:with-changelog": "cd feature-libs/my-account && release-it && cd ../..",
 ```
 
-And replace `myaccount` and `my-account` instances with the name of yours lib.
+And replace `asm` instances with the name of yours lib.
```
Comment (marlass) [inline docs/libs/creating-lib.md:232]: Shouldn't in this case example be adjusted for the asm lib?

File: feature-libs/checkout/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "spartacus",
+    "framework",
+    "storefront",
+    "checkout"
+  ],
+  "homepage": "https://github.com/SAP/spartacus",
+  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/checkout",
+  "license": "Apache-2.0",
+  "scripts": {
+    "prepare:schematics": "yarn install",
```
Comment (marlass) [inline feature-libs/checkout/package.json]: We no longer have separate node_modules in feature libs.
The `prepare:schematics` script should be removed along with it's usage. rimraf,tsc, should be referenced from the root node_modules.

File: feature-libs/checkout/tslint.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (marlass) [inline feature-libs/checkout/tslint.json]: We now use eslint. We should no longer add tslint files

File: scripts/install/run.sh [hunk 4]
```diff
@@ -91,7 +91,7 @@ function add_b2b {
 }
 
 function add_spartacus_csr {
-    ( cd ${INSTALLATION_DIR}/${1} && ng add @spartacus/schematics@${SPARTACUS_VERSION} --overwriteAppComponent true --baseUrl ${BACKEND_URL} --occPrefix ${OCC_PREFIX} && ng add @spartacus/storefinder@${SPARTACUS_VERSION} --interactive false
+    ( cd ${INSTALLATION_DIR}/${1} && ng add @spartacus/schematics@${SPARTACUS_VERSION} --overwriteAppComponent true --baseUrl ${BACKEND_URL} --occPrefix ${OCC_PREFIX} && ng add @spartacus/storefinder@${SPARTACUS_VERSION} --interactive false && ng add @spartacus/checkout@${SPARTACUS_VERSION} --interactive false
```
Comment (marlass) [inline scripts/install/run.sh]: The way we install feature libs will be changed in few hours by this PR: #11868

File: feature-libs/checkout/package.json (hunk 2 above)
Comment (plabadie) [reply feature-libs/checkout/package.json]: Fixed.

File: feature-libs/checkout/tslint.json (hunk 3 above)
Comment (plabadie) [reply feature-libs/checkout/tslint.json]: Thanks.  Fixed.
This PR was a bit stale.  I also updated angular.json.

File: docs/libs/creating-lib.md (hunk 1 above)
Comment (plabadie) [reply docs/libs/creating-lib.md:232]: Good catch.  Fixed.
The text got messed up at some point when resolving a merge conflict.

**Review by marlass — APPROVED** [review]

**Pull request merged**